hbase_sql/pom.xml

```diff
@@ -60,6 +60,17 @@
             <artifactId>spark-avro_${scala.binary.version}</artifactId>
             <version>3.2.0</version>
         </dependency>
+	<dependency>
+		<groupId>org.apache.curator</groupId>
+		<artifactId>curator-framework</artifactId>
+		<version>2.6.0</version>
+		<exclusions>
+			<exclusion>
+				<groupId>com.google.guava</groupId>
+				<artifactId>guava</artifactId>
+			</exclusion>
+		</exclusions>
```

Comment:
should check the right versions for hdp-2.5.5 and mapr 5.2.0, Jenkins showing problems on these platforms (mem and CDH platforms were ok).